tests/app/providers/test_provider_aws.py

```diff
@@ -66,7 +66,10 @@ async def test_send_push_notification(self, mock_get_session: AsyncMock, data: d
         mock_client.publish.assert_called_once()
         assert reference_id == 'message_id'
 
-    async def test_send_push_notification_botocore_exceptions_not_retriable(self, mock_get_session: AsyncMock) -> None:
+    @pytest.mark.parametrize(('name', 'exc'), [(n, e) for n, e in botocore.exceptions.__dict__.items()])
```

Comment:
The [docs tell us](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/sns/client/publish.html) which exceptions to expect but fixing this was outside the scope of what I was aiming to do.